requirements.txt

```diff
@@ -1,3 +1,5 @@
+# usage: python3 -m pip install -r requirements.txt -U
+
```

Comment:
No need to add this, that's basic information that anyone running from sources should know or otherwise it's on our wiki.

Reply:
I have to google it each time. So I would like to keep it, if OK

Reply:
No, sorry. Please remove.
If anywhere, it should be in the `README.md`.